apps/meteor/client/omnichannel/tags/AutoCompleteTagsMultiple.tsx

```diff
@@ -1,43 +1,37 @@
-import type { PaginatedMultiSelectOption } from '@rocket.chat/fuselage';
 import { PaginatedMultiSelectFiltered } from '@rocket.chat/fuselage';
 import { useDebouncedValue } from '@rocket.chat/fuselage-hooks';
+import type { ComponentProps } from 'react';
 import { memo, useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 
-import { useRecordList } from '../../hooks/lists/useRecordList';
-import { AsyncStatePhase } from '../../hooks/useAsyncState';
-import { useTagsList } from '../../hooks/useTagsList';
+import { useTagsList } from '../../components/Omnichannel/hooks/useTagsList';
 
-type AutoCompleteTagsMultipleProps = {
-	id?: string;
-	value?: PaginatedMultiSelectOption[];
-	onlyMyTags?: boolean;
-	onChange?: (value: PaginatedMultiSelectOption[]) => void;
+type AutoCompleteTagsMultipleProps = Omit<
+	ComponentProps<typeof PaginatedMultiSelectFiltered>,
+	'filter' | 'setFilter' | 'options' | 'endReached' | 'renderItem'
+> & {
 	department?: string;
 	viewAll?: boolean;
 };
```

Comment:
Why more props available for the component instead of keeping it restricted?

Reply:
My idea here was to allow the same level of customization the underlying component provides, omitting only the properties that would compromise the behavior that this autocomplete specialization is trying to facilitate.